edi/tests/test_edi_mixin.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Creu Blanca
+# @author: Enric Tobella
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import base64
+
+import mock
+from odoo_test_helper import FakeModelLoader
+
+from .common import EDIBackendCommonTestCase
+
+
+class EDIBackendTestCase(EDIBackendCommonTestCase, FakeModelLoader):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        # Load fake models ->/
+        cls.loader = FakeModelLoader(cls.env, cls.__module__)
+        cls.loader.backup_registry()
+        from .models import EdiExchangeConsumerTest
+
+        cls.loader.update_registry((EdiExchangeConsumerTest,))
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.loader.restore_registry()
+        super().tearDownClass()
+
+    def setUp(self):
+        super().setUp()
+        self.new_record = (
```

Comment:
this should stay in setUpClass. Never re-create records for all tests if not really required (which is almost never) :wink: